packages/core/content-manager/shared/contracts/history-versions.ts

```diff
@@ -0,0 +1,14 @@
+import type { Entity, UID } from '@strapi/types';
+
+/**
+ * Unlike other Content Manager contracts, history versions can't be created via the API,
+ * but only by the history service. That's why we export the create type directly here.
+ */
+export interface CreateHistoryVersion {
+  contentType: UID.ContentType;
+  relatedDocumentId: Entity.ID;
+  locale: string;
+  status: 'draft' | 'published' | 'modified' | null;
```

Comment:
We probably will not expose the 'modified' version in more places than the content manager api result. Is it planed to store the `modified` status in the history?

Reply:
nvm, just saw your comment in the history service!

Reply:
Yes we'll need to show that a history version had the modified status in the UI, even though it's a virtual status, so we're storing it on purpose.

We'll probably have to check for a matching published version when saving to get that status